<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Двумерные аффинные преобразования</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="LabOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вертеть по X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вертеть по Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вертеть по Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mirrorX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отзеркалитьпо  X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mirrorY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отзеркалитьпо  Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_top">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bottom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scaleUpX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Увеличить по X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scaleDownX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Уменьшить по X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scaleDownY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Уменьшить по Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scaleUpY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Увеличить по Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_default">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуть</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">labopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
